src/simulators/overview_simulator.ts

```diff
@@ -33,9 +34,10 @@ export class OverviewSimulator implements Simulator {
 
     const robotPosition = this.figureEight(t, fieldLength / 2, fieldWidth / 2)
 
-    const ballWorldPosition = this.figureEight(t, fieldLength / 4, fieldWidth / 4)
+    const ballPosition = this.figureEight(t, fieldLength / 4, fieldWidth / 4)
 
-    const robotHeading = ballWorldPosition.clone().subtract(robotPosition).normalize()
+    const robotHeading = ballPosition.clone().subtract(robotPosition).normalize()
+    const robotAngle = Math.atan2(robotHeading.y, robotHeading.x)
```

Comment:
Please add this comment
`// TODO (Annable): Add helper for getting the angle for a unit vector.`